roles/keytools/templates/gen-openssl-certs.sh.j2

```diff
@@ -0,0 +1,21 @@
+mkdir -p {{ openssl_backup_path }}
```

Comment:
Could you also add a shebang and exit this script upon error?

Reply:
Done.